src/Template/Reviews/of.ctp

```diff
@@ -26,6 +26,12 @@
  */
 use App\Model\CurrentUser;
 
+
+if (CurrentUser::isMember()) {
+    $this->Html->script('/js/services/list-data.srv.js', array('block' => 'scriptBottom'));
```

Comment:
You don't need this service. (It loads the selectable lists for a sentence when the user wants to add a sentence to a list.)